src/sass/theme/_datePicker.scss

```diff
@@ -0,0 +1,45 @@
+.datepicker {
+  & .btn:hover {
+    transform: scale(1);
+  }
+
+  &  button.btn.btn-default {
+    background-color: transparent;
+  }
+
+  & button.btn.btn-default.active {
+    background-color: $info;
+    //color: white;
+  }
+
+  & button.btn.active span.ng-binding.text-info {
+    color: white;
+  }
+}
+
+.uib-datepicker-wrap {
+  display:inline-block;
+  min-height:270px;
+}
+
+.uib-datepicker span.ng-binding.text-muted {
+  color: $default-text;
+}
+.uib-datepicker-popup {
+  & .ng-binding {
+    color: rgba($progress-background, 1);
+  }
+
+  & span.ng-binding.text-muted {
+    color: rgba($progress-background, 1);
+  }
+
+  & .glyphicon {
+    color: rgba($progress-background, 1);
+  }
+
+  & {
```

Comment:
This line is useless. You can put these styles right under `.uib-datepicker-popup {` line